app.js

```diff
@@ -60,23 +75,202 @@ nameForm.addEventListener('submit', (event) => {
+                    userPoints.innerHTML = appState.points.user;
+                    userWonRound();
+                } else {
+                    appState.points.computer++;
+                    computerPoints.innerHTML = appState.points.computer;
+                    computerWonRound();
+                }
+                break;
+            default:
+                appState.computerSelectedItem = scissors;
+                computerElements.scissorsElement.classList.add('item-visible');
+
+                if(appState.userSelectedItem === scissors) {
+                    deadHeat();
+                } else if(appState.userSelectedItem === rock) {
+                    appState.points.user++;
+                    userPoints.innerHTML = appState.points.user;
+                    userWonRound();
+                } else {
+                    appState.points.computer++;
+                    computerPoints.innerHTML = appState.points.computer;
+                    computerWonRound();
+                }
+        }
+        
+    }, 1500);
+}
+
+function showNextRoundStatement(msg) {
+    roundModal.modal.classList.remove('invisible');
+    roundModal.msg.innerHTML = msg;
+}
+
+function deadHeat() {
+    setTimeout( () => {
+        showNextRoundStatement('No one won this round...');
+        nextRound();
+    }, 2000);
+}
+
+function userWonRound() {
+    setTimeout( () => {
+        showNextRoundStatement('You won this round...');
+        nextRound();
+    }, 2000);
+}
+
+function computerWonRound() {
+    setTimeout( () => {
+        showNextRoundStatement('Computer won this round...');
+        nextRound();
+    }, 2000);
+}
+
+roundModal.btn.addEventListener('click', () => {
+    roundModal.modal.classList.add('invisible');
+})
+
+function disablingBtn() {
+    playerElements.paperBtn.disabled = true;
+    playerElements.rockBtn.disabled = true;
+    playerElements.scissorsBtn.disabled = true;
+
+    submitPlayerSelectionBtn.disabled = true;
+}
+
+function btnActive() {
+    playerElements.paperBtn.disabled = false;
+    playerElements.rockBtn.disabled = false;
+    playerElements.scissorsBtn.disabled = false;
+
+    submitPlayerSelectionBtn.disabled = false;
+}
+
+function nextRound() {
+        appState.round++;
+        round.innerHTML = appState.round
+
+        clearSelections();
+        hideElements();
+
+        btnActive();
+        
+        if(appState.points.computer === 3 || appState.points.user === 3) {
+            endGameModal.modal.classList.remove('invisible');
+            whoWonMsg(winner() );
+            endGameModal.playAgain.addEventListener('click', () => {
+                clearAppStateWithoutName();
+                endGameModal.modal.classList.add('invisible');
+                roundModal.modal.classList.add('invisible');
+                inputName.value = null;
+                
+            });
+            endGameModal.playAgainSwitchName.addEventListener('click', () => {
+                clearAppState();
```

Comment:
if you do a reload of the page, there is no need for `clearAppState`. I think we should rather avoid reloading the page to be modern ;) But this point we will discuss together